<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveletWidget</class>
 <widget class="QWidget" name="WaveletWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>509</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string>QWidget#statusBar {
	border-top: 1px solid #FFFFFF;
	background-color: #DDDDDD;
}
QWidget#participants {
	background-color: rgb(201, 226, 252);
}
QWidget#toolbar {
	background-image: url(:/gui/gfx/wavelet-toolbar-bg.png);
	background-repeat: repeat-x;
	background-position: left center;
	border-top: 1px solid #B8C6D9;
	border-bottom: 1px solid #B8C6D9;
}
QWidget#toolbar &gt; QToolButton {
	height: 16px;
	border: none;
	padding: 3px 1px;
	margin-left: 2px;
}
QWidget#toolbar &gt; QToolButton:hover {
	background-image: url(:/gui/gfx/wavelet-toolbar-hover.png);
	background-repeat: repeat-x;
	background-position: left center;
	border: 1px solid #B8C6D9;
	padding: 2px 0;
}
QWidget#toolbar &gt; QToolButton:pressed {
	background-image: url(:/gui/gfx/wavelet-toolbar-active.png);
	padding: 3px 1px 1px 1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="participants" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="participantsLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="cmdAddParticipant">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add participant</string>
        </property>
        <property name="icon">
         <iconset resource="../main.qrc">
          <normaloff>:/gui/icons/list-add.png</normaloff>:/gui/icons/list-add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>592</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="cmdReply">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Reply</string>
        </property>
        <property name="icon">
         <iconset resource="../main.qrc">
          <normaloff>:/gui/icons/reply-all.png</normaloff>:/gui/icons/reply-all.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>643</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWebView" name="content">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="statusBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="statusMessage"/>
      </item>
      <item>
       <widget class="QLabel" name="documentStatus">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections/>
</ui>
